src/haddock/core/mandatory.yaml

```diff
@@ -0,0 +1,19 @@
+run_dir:
+  type: dir
+  title: Run directory
+  short: Folder to store the HADDOCK3 run
+  long: The new folder that will be created to save the HADDOCK3 run
+  group: mandatory
```

Comment:
This seem double. 
Also by specifying a group the builder will move the parameter into a collapsible group and not as the first parameter.

run_dir and molecules do not have a default value that makes them required for the builder already.

Can the group be removed?
```suggestion
```

Reply:
Yes, I can remove the group.